File: webapp-net/Core/Models/Entity/Topic.cs
using Sdl.Web.Common.Models;
using System;
using Newtonsoft.Json;

namespace Sdl.Web.Modules.Core.Models
{
    /// <summary>
    /// Topic Entity
    /// </summary>
    [SemanticEntity(Vocab = SchemaOrgVocabulary, EntityName = "Topic", Prefix = "s", Public = true)]
    [Serializable]
    public class Topic : EntityModel
    {
        [SemanticProperty("topicBody")]
        [JsonProperty(PropertyName = "topicBody")]
        public RichText TopicBody { get; set; }

        [SemanticProperty("topicTitle")]
        [JsonProperty(PropertyName = "topicTitle")]
        public string TopicTitle { get; set; }
    }
}

File: webapp-java/dxa-module-dynamic-documentation/src/main/java/com/sdl/dxa/modules/dd/models/Topic.java
package com.sdl.dxa.modules.dd.models;

import com.sdl.webapp.common.api.model.RichText;
import com.sdl.webapp.common.api.model.entity.AbstractEntityModel;

//[SemanticEntity(Vocab = SchemaOrgVocabulary, EntityName = "Topic", Prefix = "s", Public = true)]
public class Topic extends AbstractEntityModel {
    //        [SemanticProperty("topicBody")]
//        [JsonProperty(PropertyName = "topicBody")]
    private RichText topicBody;

    //        [SemanticProperty("topicTitle")]
//        [JsonProperty(PropertyName = "topicTitle")]
    private String topicTitle;

    public RichText getTopicBody() {
        return topicBody;
    }

    public void setTopicBody(RichText topicBody) {
        this.topicBody = topicBody;
    }

    public String getTopicTitle() {
        return topicTitle;
    }

    public void setTopicTitle(String topicTitle) {
        this.topicTitle = topicTitle;
    }
}
